README.md

```diff
@@ -1,2 +1,14 @@
 # http-server
 CF 401 Python http-server Assignment
+
+Collaborators: Joseph McClenahan, Jared Scarr
+
+This app allows for a client to send an http request.
+The server will then parse that request and return either
+a 200 OK response or a 500 Error.
+
+Resourcess:
+
+Python Docs
+stackoverflow.com
+google.com
```

Comment:
This seems a bit out of date.  Update it please to reflect what the server does now.